libmozevent/mercurial.py

```diff
@@ -305,6 +307,24 @@ async def run(self):
                     "Unsupported repository", repo=build.repo_phid, build=build
                 )
 
+    def is_commit_skippable(self, build):
+        def get_files_touched_in_diff(rawdiff):
+            patched = []
+            for parsed_diff in rs_parsepatch.get_diffs(rawdiff):
+                # filename is sometimes of format 'test.txt  Tue Feb 05 17:23:40 2019 +0100'
```

Comment:
Could you point to the issue you file in rs_parsepatch?

Reply:
yup, will do.